scripts/get

```diff
@@ -60,7 +60,7 @@ download_binary() {
 
     mkdir -p "$(dirname "$OUTPUT")"
 
-    if command -v cosign2 >/dev/null; then
+    if command -v cosign >/dev/null; then
```

Comment:
Whoops, what a typo.

Reply:
I thought the binary was cosign2 :upside_down_face: